serve.sh

```diff
@@ -3,7 +3,4 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-cd out
-
-# https://docs.python.org/3/library/http.server.html#module-http.server
-python3 -m http.server 8000 --bind 127.0.0.1
+dev_appserver.py app.yaml
```

Comment:
We might as well just get rid of this file. Running `dev_appserver.py` directly in this directory will launch the local server.